### ISSUE TITLE
Can't map /etc/fileshelter.conf outside of the host...

### ISSUE THREAD

**Issue by thestraycat** [paulgalow/fileshelter-docker#1]
Running this on unraid and can't seem to map the /etc/fileshelter.conf file outside of the container to stay persistant. Can it be done?

**Comment by paulgalow** [comment]
I'm afraid, I haven't had any chance to work with unRAID. Does ist support Docker Compose or just the Docker daemon, i.e. Docker Engine? This workflow is using Docker Compose. If Compose is not supported you might be able to run the FileShelter Docker image separately and exposing the fileshelter.conf [file directly](https://docs.docker.com/engine/reference/commandline/run/#options).

**Comment by thestraycat** [comment]
Yeah thats what i've tried... (exposing the fileshelter.conf file) but for some reason it errors out the container.

**Comment by thestraycat** [comment]
I was thinking ... would it not be easier to save out the fileshelter.conf file to /var/fileshelter/fileshelter.conf? At the moment it lives in /etc/fileshelter.conf. I can pass /var out it seems... not sure why explicitly passing /etc/fileshelter.conf shouldnt work though?! Do you use the UID and GID variables at all?

**Comment by paulgalow** [comment]
I guess you could do that, yes. I'm just sticking to the "convention" of saving configuration files in /etc/. But I've had issues like you mentioned before. Sometimes this can be due to permissions or similar UID / GID values on the host and container side. I'm using 1000 for UID and GID in the application container so maybe this collides with your local user? Perhaps changing ownership of your host folder to a different user might solve this?
